src/appkit.cs

```diff
@@ -9345,6 +9666,14 @@ public interface NSLayoutGuide : NSCoding, NSUserInterfaceItemIdentification
 		[Export ("centerYAnchor", ArgumentSemantic.Strong)]
 		NSLayoutYAxisAnchor CenterYAnchor { get; }
 #endif // XAMCORE_2_0
+
+		[Mac (10, 12)]
+		[Export ("hasAmbiguousLayout")]
+		bool HasAmbiguousLayout { get; }
+
+		[Mac (10,12)]
+		[Export ("constraintsAffectingLayoutForOrientation:")]
+		NSLayoutConstraint[] GetConstraints (NSLayoutConstraintOrientation orientation);
```

Comment:
Better read docs, it might be better to use `GetConstraintsAffectingLayout` is it's possible that other API (existing or future) could return constraints *not* affecting layout (or all of them).